Engineering thesis/rozdzialy/Bibliografia.tex

```diff
@@ -0,0 +1,51 @@
+\begin{thebibliography}{1}
+\bibitem {Socket.IO} Socket.IO:
+\\
+ https://ably.com/topic/socketio
+
+\bibitem {Socket.IO} Socket.IO:
+\\
+ https://en.wikipedia.org/wiki/Socket.IO
+\bibitem {Komunikacja klient-serwer}Komunikacja klient-serwer:
+\\
+ https://vavatech.pl/technologie/architektura/Klient-Server
```

Comment:
jaki jest cel tych dwóch linków? Dlaczego wysyłasz linki do tutoriali? Tu nie ma miejsca na to

Reply:
Jest już tylko jeden link